visor/visor.go

```diff
@@ -176,12 +178,17 @@ func (v *Visor) Run(ctx context.Context) error {
 
 					numOfUpgradeStatusErrs++
 
-					v.log.Info("Still waiting for vega to start...", logging.Int("attemptLeft", maxUpgradeStatusErrs-numOfUpgradeStatusErrs))
+					attemptsLeft := maxUpgradeStatusErrs - numOfUpgradeStatusErrs
+					if numOfRestarts == 0 {
```

Comment:
This is a no show stopper but I would (since it's used in 2 places) store it in a variable called something like `isFirstStartup`. Just to make it very clear what we are looking for.